buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt

```diff
@@ -31,9 +31,9 @@ package io.spine.internal.dependency
  *
  * See [`SpineEventEngine/ProtoData`](https://github.com/SpineEventEngine/ProtoData/).
  */
-@Suppress("unused")
+@Suppress("unused", "ConstPropertyName")
 object ProtoData {
-    const val version = "0.8.0"
+    const val version = "0.9.3"
```

Comment:
We have 0.9.4 already. 

And Validation works (much faster) using 0.9.4, it is not broken. So that a proper version to use.